lib/tasks/impersonation_admin_user.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :impersonation_admin_user do
+  task :update, %i[username domain] => :environment do |_task, args|
+    update_query = if System::Database.oracle?
+                     "username := :username; email := :username || '+' || accounts.self_domain || '@' || :domain"
```

Comment:
As Hery told me, better doing it with string interpolation than naming `username` and `domain`